datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -530,14 +531,14 @@ private boolean isRazSupported(String runtime, CloudPlatform cloudPlatform) {
     }
 
     /**
-     * Medium Duty HA is only on 7.2.2 and later.  If runtime is empty, then sdx-internal call was used.
+     * Medium Duty HA is only on 7.2.7 and later.  If runtime is empty, then sdx-internal call was used.
      */
```

Comment:
**suggestion:** I don't think this is meant to be a javadoc comment, but if it is, consider using the `@param` tag.

If this isn't meant to be a java doc, in the future just be sure to leave off the second splat in `/**`